protocol/daemons/server/types/health_checker.go

```diff
@@ -83,26 +83,26 @@ func (u *healthCheckerMutableState) ReportFailure(now time.Time, err error) time
 	return now.Sub(u.mostRecentFailureStreakError.Timestamp())
 }
 
-// healthChecker encapsulates the logic for monitoring the health of a health checkable service.
+// healthChecker encapsulates the logic for monitoring the health of a health-checkable service.
 type healthChecker struct {
 	// mutableState is the mutable state of the health checker. Access to these fields is synchronized.
 	mutableState *healthCheckerMutableState
 
-	// healthCheckable is the health checkable service to be monitored.
+	// healthCheckable is the health-checkable service to be monitored.
 	healthCheckable types.HealthCheckable
 
-	// pollFrequency is the frequency at which the health checkable service is polled.
+	// pollFrequency is the frequency at which the health-checkable service is polled.
 	pollFrequency time.Duration
 
-	// timer triggers a health check poll for a health checkable service.
+	// timer triggers a health check poll for a health-checkable service.
 	timer *time.Timer
 
-	// maxAcceptableUnhealthyDuration is the maximum acceptable duration for a health checkable service to
+	// maxAcceptableUnhealthyDuration is the maximum acceptable duration for a health-checkable service to
```

Comment:
nit: can we use `maxDaemonUnhealthyDuration` to be consistent w the flag name? Feels a bit confusing to go from `maxDaemonUnhealthyDuration` to `maxAcceptableUnhealthyDuration`

Reply:
Good point!